projects/carba-detect/read_adc.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+import spidev
+import time
+import math
+import os
+import statsd
+
+spi = spidev.SpiDev()
+spi.open(0,0)
+client = statsd.StatsClient(host='docker-master.beia-consult.ro', port=8125)
+
+
+def ReadInput(channel):
```

Comment:
please use PEP8 style names for functions, constants and variables.
- functions - `function_with_long_name`
- variable - `variable_with_long_name`
- constant - `CONSTANT_WITH_LONG_NAME`